archetypes/author.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "" # Name
+image: ""
+email: ""
+social:
+  - icon : "ti-linkedin" # themify icon pack : https://themify.me/themify-icons
+    link : "#" # Linkedin
+  - icon : "ti-twitter-alt" # themify icon pack : https://themify.me/themify-icons
+    link : "#" # Twitter
+  - icon : "ti-github" # themify icon pack : https://themify.me/themify-icons
+    link : "#" # Github
+---
+
+<!--Write your function below-->
+
+
+<!--Write description below-->
```

Comment:
And here could we update it to `⟵ Write a brief 1-2 sentence bio/personal description below ⟶`